frontend/src/v5/store/drawings/drawings.selectors.ts

```diff
@@ -32,6 +32,15 @@ import { Calibration } from './drawings.types';
 
 const selectDrawingsDomain = (state): DrawingsState => state?.drawings || ({ drawingsByProjectByProject: {} });
 
+const selectRawDrawingById =  createSelector(
+	selectDrawingsDomain,
+	selectCurrentProject,
+	(_, id: string) => id,
+	(state, currentProject, id) =>
+		(state.drawingsByProject[currentProject] ?? []).find((drawing) => drawing._id === id) || {},
```

Comment:
The first part of `selectRawDrawingById` and `selectDrawings` is duplicated code and could be extracted into a new selector like `selectCurrentProjectDrawings` or `selectDrawingsByProject`
![image](https://github.com/user-attachments/assets/37f79ce1-087d-44c5-9dce-4432279a241c)
<details>
  <summary>Refactored code (you can copy paste)</summary>
  
  ```ts
const selectDrawingsByProject = createSelector(
	selectDrawingsDomain,
	selectCurrentProject,
	(state, currentProject) => state.drawingsByProject[currentProject] ?? [],
);

const selectRawDrawingById =  createSelector(
	selectDrawingsByProject,
	(_, id: string) => id,
	(drawings, id) => drawings.find((drawing) => drawing._id === id) || {},
);

export const selectDrawings = createSelector(
	selectDrawingsByProject,
	selectRevisionsByDrawing, // This selector is used here to recalculate the value after the revisions are fetched
	(drawings) => orderBy(drawings.map(fullDrawing), 'lastUpdated', 'desc'),
);

export const selectDrawingById = createSelector(
	selectDrawings,
	(_, _id) => _id,
	(drawings, _id) => drawings.find((d) => d._id === _id),
);
```

</details>